docs/sources/reference/commandline/cli.md

```diff
@@ -893,7 +904,6 @@ Creates a new container.
       --cgroup-parent=""         Optional parent cgroup for the container
       --cidfile=""               Write the container ID to the file
       --cpuset-cpus=""           CPUs in which to allow execution (0-3, 0,1)
-      --cpuset-mems=""           Memory nodes (MEMs) in which to allow execution (0-3, 0,1)
```

Comment:
Not sure either since when I tried to rebase it too, just now, those diffs still show up.  I'm not enough of a git expert to know what happened, or how to fix it, aside from extracting the latest versions of those files and applying your changes again.  Or I guess just copy-n-paste those lines that show up in the diff from master.